src/søknad/steg/4-barnasbosted/BarnetsBostedEndre.tsx

```diff
@@ -55,6 +55,8 @@ const BarnetsBostedEndre: React.FC<Props> = ({
   const [barnHarSammeForelder, settBarnHarSammeForelder] = useState<
     boolean | undefined
   >(undefined);
+  const [kjennerIkkeIdent, settKjennerIkkeIdent] = useState<boolean>(false);
```

Comment:
Bør disse useStatene ha en verdi fra søknad-staten? Iallefall `kjennerIkkeIdent` ? Usikker på den første...

Reply:
Vi kunne lagt det til, det har vi gjort i IPersonDetaljer (når man spør om mer info om samboeren og ønsker at søker oppgir enten ident/fødselsdato). Men har prøvd å unngå å tweake så mye på modellene siden jeg tenkte at det var hassle å endre på apiet. useStaten brukes for det meste til checkboxen for å skifte mellom ident feltet og datovelgeren, og her har det funket å ikke legge det inn i barn-modellen enn så lenge. :)